src/pages/watchLaterPage/WatchLater.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Navbar, HorizontalCard, Sidebar, LargeHorizontalCard} from "../../components/index";
+
+function WatchLater() {
```

Comment:
one last comment on file naming: keep things consistent
 JSX: WatchLater.JSX
 css: watchLater.css
 folder: watchLater
suggestion: you can keep same case for all to be uniform

Reply:
yaah I will do that